reactioncommerce/reaction-next-starterkit
# #443: PDP: Remove Tags display

Resolves #442
Impact: **minor**
Type: **feature**

## Issue
Tags need to be removed from PDP to prepare for incoming changes to Tag logic. A Tag will be updated so that a Tag can be marked as inactive, for example, and not all Tags will have enabled live links. Because of this, we want to remove Tags from being displayed on PDP by default. 

## Solution
- Removes `TagGrid` display component from PDP
- Deletes `TagGrid` component entirely
- Updates snapshot tests for PDP

## Breaking changes
None

## Screenshots

_Given a product with tags in Admin_:
<img width="1440" alt="screen shot 2018-12-04 at 12 41 05 pm" src="https://user-images.githubusercontent.com/3673236/49471587-228a3880-f7c2-11e8-9265-dc59dbf41444.png">

_No tags should show up_:
<img width="1440" alt="screen shot 2018-12-04 at 12 40 58 pm" src="https://user-images.githubusercontent.com/3673236/49471583-21590b80-f7c2-11e8-8bec-f743a93aaec1.png">


## Testing
1. Add tags to a product in Admin
2. View the product in Starter-Kit
3. See that the tags are not shown.
